packages/cloudflare/src/index.ts

```diff
@@ -130,6 +130,14 @@ export default function createIntegration(args?: Options): AstroIntegration {
 							// Allows imports of '.wasm', '.bin', and '.txt' file types
 							cloudflareModulePlugin,
 						],
+						ssr: {
+							external: ['cloudflare:sockets'],
```

Comment:
```suggestion
							external: ['cloudflare:sockets', 'cloudflare:workers'],
```

https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/rpc/